docs/csharp/programming-guide/unsafe-code-pointers/pointer-types.md

```diff
@@ -109,7 +109,7 @@ Console.ReadLine();
   
  A pointer can be `null`. Applying the indirection operator to a null pointer causes an implementation-defined behavior.  
   
- Be aware that passing pointers between methods can cause undefined behavior. Examples are returning a pointer to a local variable through an Out or Ref parameter or as the function result. If the pointer was set in a fixed block, the variable to which it points may no longer be fixed.  
+ Be aware that passing pointers between methods can cause undefined behavior. Examples are return a pointer to a local variable through an `in`, `out` or `ref` parameter or as the function result. If the pointer was set in a fixed block, the variable to which it points may no longer be fixed.  
```

Comment:
the original "returning" is preferable, though the sentence could use some rewriting, such as, "Consider a method that returns a pointer..."